repos/system_upgrade/el7toel8/actors/multipathconfupdate/libraries/library.py

```diff
@@ -0,0 +1,249 @@
+import re
+
+from leapp.libraries.common import multipathutil
+
+_bool_options = {"retain_attached_hw_handler": True, "detect_prio": True,
+                 "detect_path_checker": True, "reassign_maps": False}
+
+_exist_options = ("hw_str_match", "ignore_new_boot_devs",
+                  "new_bindings_in_boot", "unpriv_sgio")
+
+_ovr_options = ("hardware_handler", "pg_timeout")
+
+
+class _QueueIfNoPathInfo(object):
+    def __init__(self, line, value):
+        self.line = line
+        self.value = value
+        self.has_no_path_retry = False
+
+
+def _nothing_to_do(config):
+    # pylint: disable=too-many-boolean-expressions,too-many-return-statements
+    if config.default_path_checker and config.default_path_checker != "tur":
+        return False
+    if config.default_retain_hwhandler is False or \
+            config.default_detect_prio is False or \
+            config.default_detect_checker is False or \
+            config.reassign_maps is True:
```

Comment:
About the too many boolean expressions:
```suggestion
    if not all((config.default_retain_hwhandler,
                config.default_detect_prio,
                config.default_detect_checker)) or config.reassign_maps:
```